pkg/controller/kibana/controller.go

```diff
@@ -209,6 +220,24 @@ func handleConfigMap(meta metav1.Object) bool {
 	return false
 }
 
+// handlePod returns true if metaname contains a registered kibana name as substring
+func handlePod(meta metav1.Object) bool {
+
+	// iterate over registeredKibanas that match the namespace
+	namespace := meta.GetNamespace()
+
+	registeredKibanas.mux.Lock()
+	defer registeredKibanas.mux.Unlock()
+
+	for _, kibana := range registeredKibanas.registered {
+		if kibana.Namespace == namespace {
+			return strings.Contains(meta.GetName(), kibana.Name)
```

Comment:
`strings.Contains` or should we just return `meta.GetName() == kibana.Name` ?

Reply:
Pod names include the hash given from their deployment, thus `kibana (CR) != kibana-deadbeef (Pod)`